spec/models/elastic_news_item_spec.rb

```diff
@@ -224,18 +229,41 @@
             body: 'random text here',
             contributor: 'President',
             publisher: 'Briefing Room',
-            subject: 'Economy')
-          ElasticNewsItem.commit
+            subject: 'Economy'
+          }
         end
 
-        it 'should do downcasing and ASCII folding only' do
-          appropriate_stemming = ['superknuller', 'woche']
-          appropriate_stemming.each do |query|
-            expect(ElasticNewsItem.search_for(q: query, rss_feeds: [blog], language: affiliate.indexing_locale, title_only: true).total).to eq(1)
+        context 'when affiliate locale is not one of the custom indexed languages' do
+          before do
+            affiliate.update!(locale: 'kl')
+            NewsItem.create!(news_item_params)
+            ElasticNewsItem.commit
+          end
+
+          it 'does downcasing and ASCII folding only' do
+            appropriate_stemming = ['superknuller', 'woche']
+            appropriate_stemming.each do |query|
```

Comment:
I was re-using an existing spec, but I realized that testing the array isn't necessary. `'woche'` is redundant, because `'superknuller'` tests both the downcasing and ascii folding. So I'm going to remove the array and clean up those specs a bit more.